drive_config.yaml.template

```diff
@@ -0,0 +1,7 @@
+# This file defines which folders you want Google Drive import/export serivce
+# to import files from and export translated files back.
+language_folders:
+  en:
+    id-of-google-drive-en-folder
+  es:
```

Comment:
Should we document all the places that need to be changed for each language we add?

Reply:
This is the only place for google docs (and adding the folder in GDrive), but in general that's a good idea